treeherder/config/settings.py

```diff
@@ -439,5 +439,11 @@
 # Only generate alerts for data newer than this time in seconds in perfherder
 PERFHERDER_ALERTS_MAX_AGE = timedelta(weeks=2)
 
+# PerfSheriff Bot settings
+MAX_BACKFILLS_PER_PLATFORM = {
+    'linux': 200,
```

Comment:
Do we need to add the settings for `windows` platform?

Reply:
For this quarter we've planned to enable this only on Linux.